src/odemis/acq/move.py

```diff
@@ -370,16 +373,19 @@ def getTargetPosition(self, target_pos_lbl: int) -> Dict[str, float]:
                 if current_position == LOADING:
                     # if at loading and fm is pressed, choose grid1 by default
                     sem_grid1_pos = stage_md[model.MD_SAMPLE_CENTERS][POSITION_NAMES[GRID_1]]
+                    sem_grid1_pos.update(stage_md[model.MD_FAV_SEM_POS_ACTIVE])
                     fm_target_pos = self._transformFromSEMToMeteor(sem_grid1_pos)
                 elif current_position == SEM_IMAGING:
                     fm_target_pos = self._transformFromSEMToMeteor(self.stage.position.value)
                 end_pos = fm_target_pos
         elif current_position == FM_IMAGING:
             if target_pos_lbl == GRID_1:
                 sem_grid1_pos = stage_md[model.MD_SAMPLE_CENTERS][POSITION_NAMES[GRID_1]]
+                sem_grid1_pos.update(stage_md[model.MD_FAV_SEM_POS_ACTIVE])
```

Comment:
Are these changes only required for TFS and not for Tescan and Zeiss?

Reply:
Yes, the check is only supported in TFS (for now)

Reply:
This support is already there for Tescan and Zeiss, my bad. The order followed there is updating the angle and then the grid centre coordinates, I think both give the same results. But can we make it consistent for all i.e. TFS, Tescan and zeiss regarding the above update?

Reply:
Yep, you are correct its a different order for each manufacturer. Will update